datafusion/ffi/src/udaf/accumulator_args.rs

```diff
@@ -63,11 +63,11 @@ impl TryFrom<AccumulatorArgs<'_>> for FFI_AccumulatorArgs {
             WrappedSchema(FFI_ArrowSchema::try_from(args.return_field.as_ref())?);
         let schema = WrappedSchema(FFI_ArrowSchema::try_from(args.schema)?);
 
-        let codec = DefaultPhysicalExtensionCodec {};
+        let mut codec = DefaultPhysicalExtensionCodec {};
```

Comment:
Is there a plan @timsaucer to make this configurable so we can use our own codec?  It’s likely covered in other PRs, is it?